### DISCUSSION TITLE
Academic article (R Markdown) as chapter in Quarto book

### DISCUSSION THREAD

**Discussion by bbartholdy** [quarto-dev/quarto-cli#1635] [Q&A] [answered]
Hi all,

This is probably a pretty niche use-case for Quarto, but I'm hoping someone will have an answer. 

I'm trying to include an academic article as a book chapter (the book is my PhD dissertation). The article is included as a git submodule, where I would like to use the `{{< include >}}` shortcode to import the article (avoid copy-paste and make sure any changes I make to the article are changed in the book). The article is in a .Rmd file where the section headings are level-1 headings (required for the article).

**The issue:**

When I include the .Rmd article file within the .qmd chapter file, i.e. `{{< include path/to/paper.Rmd >}}`, the headings of the PDF output are off; each section of the article gets its own chapter in the book.
I have used `shift-heading-level-by: 1` in the YAML header of the chapter .qmd file, but it only seems to work for the HTML output, not the PDF output. Is there a reason it only works for HTML, and is there a better way to achieve what I'm trying to do?

**Comment by cscheid** [comment, marked as answer]
The issue, I suspect, is that `shift-heading-level-by` is a _global_ yaml option (`include` is a pretty blunt tool: it inlines all the code, and the rest of quarto doesn't know that the options you set are only meant for that document). In order for what you're suggesting to work, I think you'd have to do more clever things.

The Lua filter route is to wrap your include call in a div block with a particular class, then have your lua filter find that block, walk its children, and replace headings manually with headings that are one level lower.

**Reply by bbartholdy** [reply]
That's what I was afraid of... haven't worked with lua before :grimacing:
